packages/odyssey-react-mui/src/themes/odyssey/components.ts

```diff
@@ -659,6 +659,73 @@ export const components: ThemeOptions["components"] = {
       },
     },
   },
+  MuiTab: {
+    defaultProps: {
+      iconPosition: "start",
+    },
+    styleOverrides: {
+      root: ({ theme, ownerState }) => ({
+        maxWidth: `calc(${theme.mixins.maxWidth} / 2)`,
+        minWidth: "unset",
+        minHeight: "unset",
+        padding: `${theme.spacing(4)} 0`,
+        lineHeight: theme.typography.body.lineHeight,
+        overflow: "visible",
+        ...(ownerState.selected == true && {
+          color: theme.palette.text.primary,
+        }),
+        ...(ownerState.textColor === "inherit" && {
+          color: "inherit",
+          opacity: 1,
+        }),
+        ...(ownerState.wrapped && {
+          fontSize: theme.typography.caption.fontSize,
+          lineHeight: theme.typography.caption.lineHeight,
+        }),
+        "&:hover": {
+          color: theme.palette.primary.main,
+        },
+        "&:focus-visible::before, &.Mui-focusVisible::before": {
+          content: "''",
```

Comment:
We specifically want a `''` in the content?

Reply:
We need `content` for the pseudo-element to appear, but we want it to be empty.

The parent `""` indicates that this is a string and the nested `''` is the empty string we generate in CSS.